cluster/images/hyperkube/static-pods/master-multi.json

```diff
@@ -57,18 +59,19 @@
       "name": "scheduler",
       "image": "REGISTRY/hyperkube-ARCH:VERSION",
       "command": [
-              "/hyperkube",
-              "scheduler",
-              "--master=127.0.0.1:8080",
-              "--v=2"
-        ]
+        "/hyperkube",
+        "scheduler",
+        "--master=127.0.0.1:8080",
+        "--leader-elect=true",
+        "--v=2"
+      ]
     },
     {
       "name": "setup",
       "image": "REGISTRY/hyperkube-ARCH:VERSION",
       "command": [
-              "/setup-files.sh",
-              "IP:10.0.0.1,DNS:kubernetes,DNS:kubernetes.default,DNS:kubernetes.default.svc,DNS:kubernetes.default.svc.cluster.local"
+        "/setup-files.sh",
+        "IP:10.0.0.1,DNS:kubernetes,DNS:kubernetes.default,DNS:kubernetes.default.svc,DNS:kubernetes.default.svc.cluster.local"
       ],
       "volumeMounts": [
         {
```

Comment:
@zreigz The problem is here with your apiserver not starting. Somehow `/data` is still left here, but t should certainly be `/srv/kubernetes`